app/server/src/main/scala/org/bitcoins/server/Main.scala

```diff
@@ -0,0 +1,131 @@
+package org.bitcoins.server
+
+import org.bitcoins.rpc.config.BitcoindInstance
+import org.bitcoins.node.models.Peer
+import org.bitcoins.core.util.BitcoinSLogger
+import org.bitcoins.rpc.client.common.BitcoindRpcClient
+import akka.actor.ActorSystem
+import scala.concurrent.Await
+import scala.concurrent.duration._
+import org.bitcoins.wallet.config.WalletAppConfig
+import org.bitcoins.node.config.NodeAppConfig
+import java.nio.file.Files
+import scala.concurrent.Future
+import org.bitcoins.wallet.LockedWallet
+import org.bitcoins.wallet.Wallet
+import org.bitcoins.wallet.api.InitializeWalletSuccess
+import org.bitcoins.wallet.api.InitializeWalletError
+import org.bitcoins.node.SpvNode
+import org.bitcoins.chain.blockchain.ChainHandler
+import org.bitcoins.chain.models.BlockHeaderDAO
+import org.bitcoins.chain.config.ChainAppConfig
+import akka.http.scaladsl.server.Directives._
+import org.bitcoins.wallet.api.UnlockedWalletApi
+import org.bitcoins.wallet.api.UnlockWalletSuccess
+import org.bitcoins.wallet.api.UnlockWalletError
+import org.bitcoins.node.networking.peer.DataMessageHandler
+import org.bitcoins.node.SpvNodeCallbacks
+
+object Main
+    extends App
+    // TODO we want to log to user data directory
+    // how do we do this?
+    with BitcoinSLogger {
+  implicit val conf = {
+    // val custom = ConfigFactory.parseString("bitcoin-s.network = testnet3")
+    BitcoinSAppConfig()
```

Comment:
I don't really like this syntax, it would be nicer if we could have something like `fromDefaultConf()` or `fromDefaultDatadir()` rather than just an empty param that looks like magic is happening when you read code

Reply:
I agree. File an issue and tackle in a separate PR? Maybe even `default()` would be nice name

Reply:
https://github.com/bitcoin-s/bitcoin-s/issues/588